contracts/FiatToken.cdc

```diff
@@ -1,1017 +1,1000 @@
+	access(all)	let BlocklisterCapReceiverPubPath: PublicPath
+	access(all)	let BlocklisterUUIDPubPath: PublicPath
+	access(all)	let BlocklisterPubSigner: PublicPath
+	access(all)	let PauseExecutorStoragePath: StoragePath
+	access(all)	let PauserStoragePath: StoragePath
+	access(all)	let PauserCapReceiverPubPath: PublicPath
+	access(all)	let PauserUUIDPubPath: PublicPath
+	access(all)	let PauserPubSigner: PublicPath
+	access(all)	let AdminExecutorStoragePath: StoragePath
+	access(all)	let AdminStoragePath: StoragePath
+	access(all)	let AdminCapReceiverPubPath: PublicPath
+	access(all)	let AdminUUIDPubPath: PublicPath
+	access(all)	let AdminPubSigner: PublicPath
+	access(all)	let OwnerExecutorStoragePath: StoragePath
+	access(all)	let OwnerStoragePath: StoragePath
+	access(all)	let OwnerCapReceiverPubPath: PublicPath
+	access(all)	let OwnerUUIDPubPath: PublicPath
+	access(all)	let OwnerPubSigner: PublicPath
+	access(all)	let MasterMinterExecutorStoragePath: StoragePath
+	access(all)	let MasterMinterStoragePath: StoragePath
+	access(all)	let MasterMinterCapReceiverPubPath: PublicPath
+	access(all)	let MasterMinterUUIDPubPath: PublicPath
+	access(all)	let MasterMinterPubSigner: PublicPath
+	access(all)	let MinterControllerStoragePath: StoragePath
+	access(all)	let MinterControllerUUIDPubPath: PublicPath
+	access(all)	let MinterControllerPubSigner: PublicPath
+	access(all)	let MinterStoragePath: StoragePath
+	access(all)	let MinterUUIDPubPath: PublicPath
+	
+	// ------- FiatToken States / Variables -------
+	access(all)	let name: String
+	
+	access(all)	var version: String
+	
+	// Set to true if the contract is paused
+	access(all)	var paused: Bool
+	
+	// The token total supply
+	access(all)	var totalSupply: UFix64
+	
+	// Blocked resources dictionary {resourceId: Block Height}
+	access(all)	let blocklist: {UInt64: UInt64}
+	
+	// Managed minters dictionary {MinterController: Minter}
+	access(all)	let managedMinters: {UInt64: UInt64}
+	
+	// Minter allowance dictionary {Minter: Allowance}
+	access(all)	let minterAllowances: {UInt64: UFix64}
+	
+	// ------- FiatToken Interfaces  -------
+	access(all)	resource interface ResourceId { 
+		access(all) view fun UUID(): UInt64
+	}
+	
+	access(all)	resource interface AdminCapReceiver{ 
+		access(all) fun setAdminCap(cap: Capability<&FiatToken.AdminExecutor>): Void
+	}
+	
+	access(all)	resource interface OwnerCapReceiver{ 
+		access(all) fun setOwnerCap(cap: Capability<&FiatToken.OwnerExecutor>): Void
+	}
+	
+	access(all)	resource interface MasterMinterCapReceiver{ 
+		access(all) fun setMasterMinterCap(cap: Capability<&FiatToken.MasterMinterExecutor>): Void
+	}
+	
+	access(all)	resource interface BlocklisterCapReceiver{ 
+		access(all) fun setBlocklistCap(cap: Capability<&FiatToken.BlocklistExecutor>): Void
+	}
+	
+	access(all)	resource interface PauseCapReceiver{ 
+		access(all) fun setPauseCap(cap: Capability<&FiatToken.PauseExecutor>): Void
+	}
+	
+	// ------- Capability Destruction -------
```

Comment:
Before, this function took a private path as an argument and unlinked the path. Now that private paths don't exist, we need to take the storage path to the resource that the capability was linked to and delete all capabilities for it. Since only one capability should exist at a time for these privileged resources, this should be sufficient